types/puppeteer/index.d.ts

```diff
@@ -216,36 +277,40 @@ export interface Request {
 export interface Response {
   buffer(): Promise<Buffer>;
   headers: Headers;
-  json(): Promise<object>;
+  json(): Promise<any>;
   ok: boolean;
   request(): Request;
   status: number;
   text(): Promise<string>;
   url: string;
 }
 
-export type Serializable = boolean | number | string | object;
```

Comment:
@davidkpiano I removed the `Serializable` type because using `object` instead of `any` means you are forced to cast before using the variable. 

For instance if we changed all the places the `puppeteer` docs day to use `Serializable` then the following code throws the error `Property 'innerHTML' does not exist on type 'string | number | boolean | object | ElementHandle'`:
```js
const bodyHandle = await page.$('body');
const html = await page.evaluate(body => body.innerHTML, bodyHandle);
await bodyHandle.dispose(); 
```

If we define it as `any` it compiles and runs fine.